src/imu.cpp

```diff
@@ -345,6 +345,17 @@ std::map <std::string, std::string> Imu::Info::toMap() const {
   return map;
 }
 
+uint16_t Imu::Info::getModelNumber() const {
+  // Simply convert the first contiguous group of numbers into an int
+  int modelInt = std::stoi(modelNumber.c_str());
+  // If cannot convert, default to the -25 (6324)
+  if (!modelInt)
```

Comment:
`std::stoi` throws a `std::invalid_argument` exception if no conversion could be performed so you would have to use `try-catch` for that.